chain/network/src/peer/transfer_stats.rs

```diff
@@ -59,15 +59,15 @@ impl TransferStats {
     }
 
     /// Get stats stored in `MinuteStats` struct.
-    pub(crate) fn minute_stats(&mut self, now: Instant) -> MinuteStats {
+    pub(crate) fn minute_stats(&mut self, now: time::Instant) -> MinuteStats {
         self.remove_old_entries(now);
         MinuteStats { bytes_per_min: self.total_bytes_in_events, count_per_min: self.events.len() }
     }
 
     /// Remove entries older than 1m.
-    fn remove_old_entries(&mut self, now: Instant) {
+    fn remove_old_entries(&mut self, now: time::Instant) {
         while let Some(event) = self.events.pop_front() {
-            if now.duration_since(event.instant) > TRANSFER_STATS_INTERVAL {
+            if now - event.instant > TRANSFER_STATS_INTERVAL {
```

Comment:
AFAICT time::Instant doesn't have duration_since. SG, once Instant is a newtype rather than an alias. But I still would like to support negative durations nevertheless.

Reply:
Re: flipping arguments: wdyt about "now > t + duration" instead of "now - t > duration". Would that solve mistaking order of arguments?

Reply:
Let's think about this separately then, if/when we introduce a newtyped wrapper, code lgtm right now, perhaps it's just me who always messes up subtracting times :)